docs/Getting Started/openSUSE/openSUSE Leap Root on ZFS.rst

```diff
@@ -0,0 +1,1183 @@
+     Rpm Name:         gpg-pubkey-3dbdc284-53674dd4
+
+     Do you want to reject the key, trust temporarily, or trust always? [r/t/a/?] (r): 
+
+  
+#. Install openSUSE Leap with zypper:
+
+    If you install `base` pattern, zypper will install `busybox-grep` which is masks default kernel package.
+    Thats why I recommend you to install `enhanced_base` pattern, if you're new in openSUSE. But in `enhanced_base`, bloats 
+    can annoy you, while you want to use it openSUSE on server. So, you need to select 
+
+
+    a. Install base packages of openSUSE Leap with zypper (Recommended for server)::
+
+        zypper --root /mnt install -t pattern base
+
+
+    b. Install enhanced base of openSUSE Leap with zypper (Recommended for desktop)::
+
+        zypper --root /mnt install -t pattern enhanced_base
+        
+     
+#. Install openSUSE zypper package system into chroot::
+
+     zypper --root /mnt install zypper
+
+#. Recommended: Install openSUSE yast2 system into chroot::
+
+   It will make easier to configure network and other configurations for beginners.
+   
+     zypper --root /mnt install yast2
+
+To install a desktop environment, see the `openSUSE wiki
+<https://en.opensuse.org/openSUSE:Desktop_FAQ#How_to_choose_a_desktop_environment.3F>`__
+
+Step 5: System Configuration
+----------------------------
+
+#. Configure the hostname:
+
+   Replace ``HOSTNAME`` with the desired hostname::
+
+     echo HOSTNAME > /mnt/etc/hostname
+     vi /mnt/etc/hosts
+
+   .. code-block:: text
+
+     Add a line:
+     127.0.1.1       HOSTNAME
+     or if the system has a real name in DNS:
+     127.0.1.1       FQDN HOSTNAME
+
+   **Hint:** Use ``nano`` if you find ``vi`` confusing.
+
+#. Copy network information::
+
+     cp /etc/resolv.conf /mnt/etc 
+
+   You will reconfigure network with yast2.
+   
+#. Bind the virtual filesystems from the LiveCD environment to the new
+   system and ``chroot`` into it::
+
+     mount --rbind /dev  /mnt/dev
+     mount --rbind /proc /mnt/proc
+     mount --rbind /sys  /mnt/sys
+     chroot /mnt /usr/bin/env DISK=$DISK bash --login
+
+   **Note:** This is using ``--rbind``, not ``--bind``.
+
+#. Configure a basic system environment::
+
+     ln -s /proc/self/mounts /etc/mtab
+     zypper refresh
+
+   Even if you prefer a non-English system language, always ensure that
+   ``en_US.UTF-8`` is available::
+
+     locale -a
+
+   Output must include that languages:
+
+   * C
+   * C.UTF-8
+   * en_US.utf8
+   * POSIX
+
+   Find yout locale from `locale -a` commands output then set it with following command.
+
+     localectl set-locale LANG=en_US.UTF-8
+
+
+#. Optional: Reinstallation for stability::
+
+   After installation it may need. Some packages may have minor errors. For that, do this if you wish. Since there is no command like dpkg-reconfigure in openSUSE,  [zypper install -f stated as a alternative for it](https://lists.opensuse.org/opensuse-factory/2009-07/msg00188.html) but it will reinstall packages.
```

Comment:
Another syntax issue here

Reply:
I'm fixing all syntax issues first then I gonna look at the changes in Debian and I will adapt it on opensuse and test it. I'm install another server Opensuse today. It would be helpful